src/test/java/teammates/ui/webapi/SubmitFeedbackResponsesActionTest.java

```diff
@@ -252,4 +320,14 @@ public void testAccessControl_instructorResultIntent_shouldThrowInvalidHttpParam
 
         verifyHttpParameterFailureAcl(submissionParams);
     }
+
+    private void verifyGiverTypeIsStudentEmail(StudentAttributes expected, FeedbackResponseData actual){
```

Comment:
I think it might be worth removing these methods since they're only called once each.

Reply:
yeah but the thing is they always have a seperate function for assertion tests